contracts/src/NativeSeiTokensERC20.sol

```diff
@@ -38,7 +38,7 @@ contract NativeSeiTokensERC20 is ERC20 {
     }
 
     function _update(address from, address to, uint256 value) internal override {
-        bool success = BankPrecompile.send(from, to, denom, value);
+        (bool success,) = BANK_PRECOMPILE_ADDRESS.delegatecall(abi.encodeWithSignature("sendFromCaller(address,string,uint256)", to, denom, value));
```

Comment:
this would make transferFrom also transfer from the msg.sender, which isn't correct

Reply:
updated so that we still support `from->to` bank send but only for whitelisted codehash